tests/test_comparison_template_lib.py

```diff
@@ -0,0 +1,154 @@
+import pandas as pd
+import pytest
+
+import splink.duckdb.duckdb_comparison_template_library as ctld
+import splink.spark.spark_comparison_template_library as ctls
+from splink.duckdb.duckdb_linker import DuckDBLinker
+from splink.spark.spark_linker import SparkLinker
+
+
+def test_duckdb_date_comparison_run():
+    ctld.date_comparison("date")
+
+
+def test_spark_date_comparison_run():
+    ctls.date_comparison("date")
+
+
+def test_datediff_levels(spark):
```

Comment:
See my adjusted tests that take advantage of `parametrize` - https://github.com/moj-analytical-services/splink/pull/1101.

I'd advise we use this format instead as it's more flexible.

Reply:
Done